django/db/models/fields/related.py

```diff
@@ -1375,6 +1376,26 @@ def _check_relationship_model(self, from_model=None, **kwargs):
 
         return errors
 
+    def _check_table_uniqueness(self, **kwargs):
+        if self.db_table is None:
+            return []
+        registered_tables = {
+            model._meta.db_table: model
+            for model in self.opts.apps.get_models(include_auto_created=True)
+            if model != self.remote_field.through
+        }
+        if self.db_table in set(registered_tables.keys()):
+            msg = 'Field is using a table that has already been registered by \'{}.{}\'.'
```

Comment:
I would stick to `%` formating for the sake of consistency with other `_check` methods.